src/utilities/variables.js

```diff
@@ -0,0 +1,3 @@
+export default {
```

Comment:
Ah theres a new variables :) My suggestion is to put this in the theme under transitions